tutorials/plot_background_filtering.py

```diff
@@ -483,15 +524,232 @@ def plot_signal(x, offset):
 # (among other things) calls :func:`mne.filter.filter_data` to actually
-# filter the data.
+# filter the data, which by default applies a zero-phase FIR filter designed
+# using :func:`scipy.signal.firwin2`. In Widmann *et al.* 2015 [7]_, they
+# suggest a specific set of parameters to use for high-pass filtering,
+# including:
+#
+#     "... providing a transition bandwidth of 25% of the lower passband
+#     edge but, where possible, not lower than 2 Hz and otherwise the
+#     distance from the passband edge to the critical frequency.”
+#
+# In practice, this means that for each high-pass value ``l_freq`` or
+# low-pass value ``h_freq`` below, you would get this corresponding
+# ``l_trans_bandwidth`` or ``h_trans_bandwidth``, respectively,
+# if the sample rate were 100 Hz (i.e., Nyquist frequency of 50 Hz):
+#
+# +------------------+-------------------+-------------------+
+# | l_freq or h_freq | l_trans_bandwidth | h_trans_bandwidth |
+# +==================+===================+===================+
+# |             0.01 |              0.01 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              0.1 |               0.1 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              1.0 |               1.0 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              2.0 |               2.0 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              4.0 |               2.0 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              8.0 |               2.0 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |             10.0 |               2.5 |               2.5 |
+# +------------------+-------------------+-------------------+
+# |             20.0 |               5.0 |               5.0 |
+# +------------------+-------------------+-------------------+
+# |             40.0 |              10.0 |              10.0 |
+# +------------------+-------------------+-------------------+
+# |             45.0 |             11.25 |              5.0  |
+# +------------------+-------------------+-------------------+
+# |             48.0 |              12.0 |              2.0  |
+# +------------------+-------------------+-------------------+
 #
-# :func:`mne.filter.filter_data` by default applies a FIR filter designed using
-# :func:`scipy.signal.firwin2`. For more information on how to use the
+# MNE-Python has adopted this definition for its high-pass (and low-pass)
+# transition bandwidth choices when using ``l_trans_bandwidth='auto'`` and
+# ``h_trans_bandwidth='auto'``.
+#
+# To choose the filter length automatically with ``filter_length='auto'``,
+# 7 times the reciprocal of the shortest transition bandwidth is used to ensure
+# decent attenuation by the stop frequency.
```

Comment:
well technically it could have good attenuation even *before* the stop frequency (thus the "by", as in at-or-before), but "at" works fine, too and is less confusing